### PR TITLE
[master<new-T604-FL] Transactional queue

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Introduce transaction termination

Pull request type
- [x] New documentation page 

Related PRs and issues
PR this doc page is related to: https://github.com/memgraph/memgraph/pull/790

### Checklist:

- [ ] I checked all content with Grammarly
- [ ] I performed a self-review of my code
- [ ] I made corresponding changes to the rest of the documentation
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors

### CODE REVIEW

**Comment by vpavicic** [comment]
Release note idea:
If you are no longer interested in the results of the query you ran, or the
procedure you built is running in an infinite loop, you can stop the
transaction with the TERMINATE TRANSACTIONS tid; query; Find out the
transaction ID with SHOW TRANSACTIONS; query.

**Comment by vpavicic** [comment]
I need to figure out where to write that you need to add extra code if you want to terminate a procedure
Check if we have a list of privileges anywhere else except security and add the new privilege to the list

**Review by kgolubic — APPROVED** [review]
Everything looks OK.

File: docs/how-to-guides/manage-user-privileges.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ We could naturally cluster those privileges into groups:
   * Privilege to read the content of Python query module files (`MODULE_READ`)
   * Privilege to modify the content of Python query modules files (`MODULE_WRITE`)
   * Privilege to connect to [Memgraph monitoring server](/reference-guide/monitoring-server.md) (`WEBSOCKET`)
+  * Privilege to show and terminate transactions `TRANSACTION_MANAGEMENT`.
```
Comment (vpavicic) [inline docs/how-to-guides/manage-user-privileges.md:81]:
```suggestion
  * Privilege to show and terminate transactions (`TRANSACTION_MANAGEMENT`).
```

**Pull request merged**